.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,33 @@
+name: Pull Request
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
```

Comment:
This is just for my testing and I believe we should remove it before or after merging so only the `pull_request_target` runs on any given PR (but that one can't run until this workflow is merged into master)